dockerfiles/jenkins/accept.json

```diff
@@ -0,0 +1,35 @@
+{
+  "public": [],
+  "private": [
+    {
+      "method": "GET",
+      "path": "/job/*",
+      "origin": "${JENKINS_URL}",
+      "auth": {
+        "scheme": "basic",
+        "username": "${JENKINS_USERNAME}",
+        "password": "${JENKINS_PASSWORD}"
+      }
+    },
+    {
+      "method": "GET",
+      "path": "/crumbIssuer/api/json",
+      "origin": "${JENKINS_URL}",
+      "auth": {
+        "scheme": "basic",
+        "username": "${JENKINS_USERNAME}",
+        "password": "${JENKINS_PASSWORD}"
+      }
+    },
+    {
+      "method": "POST",
```

Comment:
Hmm this means we'll need to fix the security var substition issue - guessing its required though for the plugin? https://app.shortcut.com/larder/story/13498/work-to-fix-the-vuln-suggested-by-pen-testing